### PR TITLE
chore: release v1.2.0

### PR DESCRIPTION
**Pull request by SophieManley03**
This pull request prepares the following release:
| Repository | Branch | Update | Change |
|---|---|---|---|
| [recommend](https://github.com/algolia/recommend) | releases/v1.2.0 | minor | [`1.1.0` → `1.2.0`](https://github.com/algolia/recommend/compare/v1.1.0...releases/v1.2.0) |

### Release Summary
This is going to be published with the following command:

| Dir | Command |
|---|---|
| packages/recommend-core | yarn publish --access public --tag latest |
| packages/recommend-js | yarn publish --access public --tag latest |
| packages/recommend-react | yarn publish --access public --tag latest |
| packages/recommend-vdom | yarn publish --access public --tag latest |

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release v1.2.0`.
<details>
<summary>See details</summary>

After that, a commit `chore: release v1.2.0` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+### Bug Fixes
+
+* **ci:** update docker image ([#60](https://github.com/algolia/recommend/issues/60)) ([b7770d7](https://github.com/algolia/recommend/commit/b7770d7e344b012085793953e184c3fc9e0cbe22))
+* **examples:** remove demo credentials ([5f8cb91](https://github.com/algolia/recommend/commit/5f8cb91c9476d71652dc2bc30cd0cbc07afd3fd4))
+
+
+### Features
+
+* **recommend:** Add hooks, methods, react components ([#65](https://github.com/algolia/recommend/issues/65)) ([c112371](https://github.com/algolia/recommend/commit/c112371abe12f6d22573322dd1c92978c9bcd62d))
+* **recommend:** Add tending for recommend-js ([#71](https://github.com/algolia/recommend/issues/71)) ([50c0475](https://github.com/algolia/recommend/commit/50c0475e4ca18eca1d4fc900fb829f119c2c51f3))
```
Comment (shortcuts) [inline CHANGELOG.md]:
```suggestion
* **recommend:** Add trending for recommend-js ([#71](https://github.com/algolia/recommend/issues/71)) ([50c0475](https://github.com/algolia/recommend/commit/50c0475e4ca18eca1d4fc900fb829f119c2c51f3))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+### Bug Fixes
+
+* **ci:** update docker image ([#60](https://github.com/algolia/recommend/issues/60)) ([b7770d7](https://github.com/algolia/recommend/commit/b7770d7e344b012085793953e184c3fc9e0cbe22))
+* **examples:** remove demo credentials ([5f8cb91](https://github.com/algolia/recommend/commit/5f8cb91c9476d71652dc2bc30cd0cbc07afd3fd4))
+
+
+### Features
+
+* **recommend:** Add hooks, methods, react components ([#65](https://github.com/algolia/recommend/issues/65)) ([c112371](https://github.com/algolia/recommend/commit/c112371abe12f6d22573322dd1c92978c9bcd62d))
```
Comment (shortcuts) [inline CHANGELOG.md]:
```suggestion
* **recommend:** Add hooks, methods and react components for recommend-react ([#65](https://github.com/algolia/recommend/issues/65)) ([c112371](https://github.com/algolia/recommend/commit/c112371abe12f6d22573322dd1c92978c9bcd62d))
```

**Review by shortcuts — APPROVED** [review]
Non blocking comments, it is just clearer to me

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+### Bug Fixes
+
+* **ci:** update docker image ([#60](https://github.com/algolia/recommend/issues/60)) ([b7770d7](https://github.com/algolia/recommend/commit/b7770d7e344b012085793953e184c3fc9e0cbe22))
+* **examples:** remove demo credentials ([5f8cb91](https://github.com/algolia/recommend/commit/5f8cb91c9476d71652dc2bc30cd0cbc07afd3fd4))
+
+
+### Features
+
+* **recommend:** Add hooks, methods and react components for recommend-react ([#65](https://github.com/algolia/recommend/issues/65)) ([c112371](https://github.com/algolia/recommend/commit/c112371abe12f6d22573322dd1c92978c9bcd62d))
+* **recommend:** Add trending for recommend-js ([#71](https://github.com/algolia/recommend/issues/71)) ([50c0475](https://github.com/algolia/recommend/commit/50c0475e4ca18eca1d4fc900fb829f119c2c51f3))
```
Comment (francoischalifour) [inline CHANGELOG.md]:
```suggestion
### Features

* **trending:** add Hooks and components for `recommend-react` ([#65](https://github.com/algolia/recommend/issues/65)) ([c112371](https://github.com/algolia/recommend/commit/c112371abe12f6d22573322dd1c92978c9bcd62d))
* **trending:** add UI functions for `recommend-js` ([#71](https://github.com/algolia/recommend/issues/71)) ([50c0475](https://github.com/algolia/recommend/commit/50c0475e4ca18eca1d4fc900fb829f119c2c51f3))
```

**Review by francoischalifour — APPROVED** [review]

**Pull request merged**